### PR TITLE
Fix JDBC trace doc for DB2

### PR DESCRIPTION
**Pull request by KyleAure**
The JDBC trace doc for DB2 is incorrect.

It should be: 
```xml
<logging traceSpecification="*=info:RRA=all:com.ibm.ws.db2.logwriter=all" ... />
<datasource ...>
  <!-- traceLevel -1 is equivalent to ALL -->
  <properties.db2.jcc traceLevel="-1"/>
</datasource>
```

### CODE REVIEW

File: modules/ROOT/pages/jdbc-tracing.adoc [hunk 1]
```diff
@@ -100,11 +100,11 @@ The following subsections contain JDBC driver specific configuration.
 [#DB2JCC]
 === DB2 JCC
 
-The DB2 JCC driver supports the `java.util.logging` library. The driver uses the `com.ibm.db2` package name in the trace specification. The following `server.xml` file example shows you the configuration to enable trace for the DB2 JCC driver and specify a custom property.
+The DB2 JCC does not support the `java.util.logging` library. This driver has it's own logging implementations which outputs directly to a file using a PrintWriter. Open Liberty provides the `com.ibm.ws.db2.logwriter` log writer name for trace specification that will configure a PrintWriter that will output JCC driver trace to trace.log. The following `server.xml` file example shows you the configuration to enable trace for the DB2 JCC driver and specify a custom property.
```
Comment (dmuelle) [inline modules/ROOT/pages/jdbc-tracing.adoc]:
```suggestion
The DB2 JCC does not support the `java.util.logging` library. This driver has its own logging implementations that output directly to a file by using a PrintWriter. Open Liberty provides the `com.ibm.ws.db2.logwriter` log writer name for trace specification that configures a PrintWriter to output JCC driver trace to the ` trace.log` file. The following `server.xml` file example shows you the configuration to enable trace for the DB2 JCC driver and specify a custom property.
```

File: modules/ROOT/pages/jdbc-tracing.adoc [hunk 2]
```diff
@@ -139,11 +139,11 @@ The Informix using JDBC driver supports the `java.util.logging` library. The dri
 [#InformixDB2JCC]
 === Informix using DB2 JCC
 
-The Informix using DB2 JCC driver supports the `java.util.logging` library. The driver uses the `com.ibm.db2` package name in the trace specification. The following `server.xml` file example shows you the configuration to enable trace for the Informix using DB2 JCC driver and specify a custom property.
+Informix using the DB2 JCC driver does not support the `java.util.logging` library. This driver has it's own logging implementations which outputs directly to a file using a PrintWriter. Open Liberty provides the `com.ibm.ws.db2.logwriter` log writer name for trace specification that will configure a PrintWriter that will output JCC driver trace to trace.log. The following `server.xml` file example shows you the configuration to enable trace for Informix using the DB2 JCC driver and specify a custom property.
```
Comment (dmuelle) [inline modules/ROOT/pages/jdbc-tracing.adoc]:
```suggestion
Informix using the DB2 JCC driver does not support the `java.util.logging` library. This driver has its own logging implementations that output directly to a file by using a PrintWriter. Open Liberty provides the `com.ibm.ws.db2.logwriter` log writer name for trace specification that configures a PrintWriter to output JCC driver trace to the `trace.log` file. The following `server.xml` file example shows you the configuration to enable trace for Informix by using the DB2 JCC driver and specify a custom property.
```

**Review by dmuelle — CHANGES_REQUESTED** [review]
a couple minor edits for grammar and style, otherwise LGTM

**Comment by KyleAure** [comment]
@dmuelle Approved commit suggestions.  Thanks for the quick review.

**Review by dmuelle — APPROVED** [review]

**Pull request merged**